webextensions/api/storage.json

```
@@ -246,6 +246,26 @@
                 }
               }
             }
+          },
+          "setAccessLevel": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/setAccessLevel",
+              "support": {
+                "chrome": {
+                  "version_added": "102"
```

```suggestion
                  "version_added": "96"
```

The API was already available in Chrome 96 - https://bugs.chromium.org/p/chromium/issues/detail?id=1185226#c53

(But they had forgotten to enable the documentation for many months/releases)